src/launchpad/size/insights/apple/alternate_icons_optimization.py

```diff
@@ -53,24 +63,31 @@ def _find_images(self, input: InsightsInput) -> List[FileInfo]:
 
         return list({img.path: img for img in images}.values())
 
-    def _preprocess_image(self, img: Image.Image, file_info: FileInfo) -> tuple[Image.Image, int, int]:
-        resized = self._resize_icon_for_analysis(img)
-
-        # Specify quality=85 for HEICs to avoid inflating already-optimized files.
-        # Without it, PIL uses a higher default quality (~95), which would incorrectly
-        # show savings for files that are already at our target quality.
-        fmt = img.format or "PNG"
-        with io.BytesIO() as buf:
-            if fmt.upper() in {"HEIF", "HEIC"}:
-                resized.save(buf, format="HEIF", quality=self.TARGET_HEIC_QUALITY)
-            else:
-                resized.save(buf, format=fmt)
-            resized_size = buf.tell()
-
-        baseline_savings = max(0, file_info.size - resized_size)
-        return resized, resized_size, baseline_savings
-
     def _resize_icon_for_analysis(self, img: Image.Image) -> Image.Image:
         return img.resize((self.IPHONE_3X_ICON_SIZE, self.IPHONE_3X_ICON_SIZE), Image.Resampling.LANCZOS).resize(
             (self.APP_STORE_ICON_SIZE, self.APP_STORE_ICON_SIZE), Image.Resampling.LANCZOS
         )
+
+    def _check_heic_conversion(
```

Comment:
the script that we show users literally does the resizing like this, so lets directly emulate that approach since it also yields the most similar outcome